### DISCUSSION TITLE
Recordings from ICRA'23 workshop

### DISCUSSION THREAD

**Discussion by gsilano** [osrf/icra2023_ros2_gz_tutorial#38] [General]
Hi! Amazing work here. I was at ICRA'23, but I missed this workshop. I was wondering, did you record it? If so, would the recording be available online? Thank you!

**Comment by mabelzhang** [comment]
Hello, the tutorial was not recorded. Because the majority of the tutorial was meant to be interactive - the 30-minute breakout group discussions and the two 1.5-hour hands-on practices, we had decided since the planning phase that we rather focus our limited resources to deliver a good in-person format, and not have a virtual portion at all.

That said, every piece of material used for the tutorial is already available in this repo. The technical portion is in the Docker image in the Dockerfile and DockerHub (see docker/README), and the slides for the 2 presentations and the 2 hands-on portions are linked from the "Quick Links" > "Slides" in the main README. All the notes produced by the community are linked from the breakout community notes as well. You are welcome to browse through the materials and ask us further questions here. Thanks!
